main.tf

```diff
@@ -1,5 +1,15 @@
 data "aws_caller_identity" "current" {}
 
+resource "aws_s3_bucket_notification" "bucket_notification" {
+  count  = var.notification_sns_arn != "" && var.notification_events != "" ? 1 : 0
```

Comment:
Are we sure that this is right? I *think* you might want to adjust the condition for `var.notification_events` so that the condition is `!= [""]` because the resource and the variable expect a list of strings.

Reply:
Hadn't noticed that. I will amend.